monitoring/ceph-mixin/prometheus_alerts.yml

```diff
@@ -460,7 +462,7 @@ groups:
       - alert: "CephNodeNetworkPacketDrops"
         annotations:
           description: "Node {{ $labels.instance }} experiences packet drop > 0.5% or > 10 packets/s on interface {{ $labels.device }}."
-          summary: "One or more NICs reports packet drops"
+          summary: "One or more NICs reports packet dropsssss"
```

Comment:
@aaSharma14 are you generating the alerts from libsonnet file using the script? It'd be good if you could test this out: https://github.com/ceph/ceph/pull/53653

Reply:
@nizamial09 , tested that PR, thanks for the fix. Fixed this nit as well